vru_use_case_scripts/README.md

```diff
@@ -0,0 +1,55 @@
+# VRU use-case scripts
+
+This directory contains scripts to help with data analysis for VRU use-cases.
+
+## `extract_rtf_data`
+
+This script takes in a Docker container log output and extracts realtime factor (RTF) data, outputting the results
+in comma-separated values (CSV) format.
+
+### Usage examples
+
+Output the CSV data directly to the terminal:
+
+```console
+docker logs carma-simulation 2>&1 | ./extract_rtf_data
+```
+
+Output the CSV data to a file:
+
+```console
+docker logs carma-simulation 2>&1 | ./extract_rtf_data > rtf_data.csv
+```
+
+Pipe the CSV data into the `plot_rtf_data` script (see below):
+
+```console
+docker logs carma-simulation 2>&1 | ./extract_rtf_data | ./plot_rtf_data
```

Comment:
Is there are reason we are not using an actual logs file for this and using carma-simulation docker logs?

Reply:
It doesn't look like MOSAIC writes that value to a log file.

Reply:
See [CDAR-814](https://usdot-carma.atlassian.net/browse/CDAR-814) for configuring MOSAIC's logs to record this data.